src/components/modals/FeedInfoTable.vue

```diff
@@ -0,0 +1,202 @@
+							</span>
+						</th>
+						<th @click="sortBy('lastModified')">
+							<span class="column-title">
+								{{ t('news', 'Last update') }}
+								<SortAscIcon v-if="sortKey === 'lastModified' && sortOrder === 1" />
+								<SortDescIcon v-if="sortKey === 'lastModified' && sortOrder !== 1" />
+							</span>
+						</th>
+						<th @click="sortBy('nextUpdateTime')">
+							<span class="column-title">
+								{{ t('news', 'Next update') }}
+								<SortAscIcon v-if="sortKey === 'nextUpdateTime' && sortOrder === 1" />
+								<SortDescIcon v-if="sortKey === 'nextUpdateTime' && sortOrder !== 1" />
+							</span>
+						</th>
+						<th :title="t('news', 'Articles per update')"
+							@click="sortBy('articlesPerUpdate')">
+							<span class="column-title">
+								APU
+								<SortAscIcon v-if="sortKey === 'articlesPerUpdate' && sortOrder === 1" />
+								<SortDescIcon v-if="sortKey === 'articlesPerUpdate' && sortOrder !== 1" />
+							</span>
+						</th>
+						<th :title="t('news', 'Error Count') "
+							@click="sortBy('updateErrorCount')">
+							<span class="column-title">
+								EC
+								<SortAscIcon v-if="sortKey === 'updateErrorCount' && sortOrder === 1" />
+								<SortDescIcon v-if="sortKey === 'updateErrorCount' && sortOrder !== 1" />
+							</span>
+						</th>
+					</tr>
+				</thead>
+				<tbody>
+					<tr v-for="feed in sortedFeeds" :key="feed.id">
+						<td>{{ feed.id }}</td>
+						<td>{{ feed.title }}</td>
+						<td>{{ formatDate(feed.lastModified/1000) }}</td>
+						<td>{{ formatDate(feed.nextUpdateTime*1000) }}</td>
+						<td>{{ feed.articlesPerUpdate }}</td>
+						<td :title="feed.lastUpdateError">
+							{{ feed.updateErrorCount }}
+						</td>
+					</tr>
+				</tbody>
+			</table>
+		</div>
+	</NcModal>
+</template>
+
+<script>
+import NcModal from '@nextcloud/vue/dist/Components/NcModal.js'
+import SortAscIcon from 'vue-material-design-icons/SortAscending.vue'
+import SortDescIcon from 'vue-material-design-icons/SortDescending.vue'
+import { mapState } from 'vuex'
+
+export default {
+	name: 'FeedInfoTable',
+	components: {
+		NcModal,
+		SortAscIcon,
+		SortDescIcon,
+	},
+	data() {
+		return {
+			sortKey: 'title',
+			sortOrder: 1,
+		}
+	},
+	computed: {
+		...mapState({
+			feeds: state => state.feeds.feeds,
+		}),
+		sortedFeeds() {
+			const sorted = this.feeds
+			if (this.sortKey) {
+				sorted.sort((a, b) => {
+					const valueA = a[this.sortKey]
+					const valueB = b[this.sortKey]
+					if (typeof valueA === 'string') {
+						return valueA.localeCompare(valueB) * this.sortOrder
+					} else {
+						return (valueA - valueB) * this.sortOrder
+					}
+				})
+			}
+			return sorted
+		},
+	},
+	methods: {
+		formatDate(timestamp) {
+			return new Date(timestamp).toLocaleDateString(undefined, {
+				day: '2-digit',
+				month: '2-digit',
+				year: 'numeric',
+				hour: '2-digit',
+				minute: '2-digit',
+				second: '2-digit',
+			})
```

Comment:
```suggestion
		formatDate(timestamp) {
			if (!timestamp) {
				return t('news', 'Not available');
			}
			return new Date(timestamp).toLocaleDateString(undefined, {
				day: '2-digit',
				month: '2-digit',
				year: 'numeric',
				hour: '2-digit',
				minute: '2-digit',
				second: '2-digit',
			})
```

I think this would be better in case nextUpdateTime is null which it can be because the admin disabled it or simply some kind of issue with that feed.

Reply:
Other than that looks great!